db/migrate/20220809145824_add_author_id_to_comments.rb

```diff
@@ -0,0 +1,6 @@
+class AddAuthorIdToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_column :comments, :AuthorId, :integer
```

Comment:
- [ ]  please it would be better to follow the convention of creating columns in the database they should be in snake_case like that: comment_counter ...etc please try to do that